Makefile

```diff
@@ -73,6 +73,13 @@ bench-async:
 		-a true \
 		-u username -p password
 
+# Vulnerability check
+
+target/nvd:
+	clojure -Xnvd check :classpath '"'"$$(clojure -Spath -A:db-h2:db-sqlite:db-postgres)"'"'
+
+check-vuln: target/nvd
```

Comment:
By default `nvd-clojure` just outputs to `target/nvd` so you don't need a phony here really

Reply:
oh nm I see what you did here